src/plugins/teleop/Teleop.qml

```diff
@@ -0,0 +1,399 @@
+        verticalAlignment: Text.AlignVCenter
+        horizontalAlignment: Text.AlignHCenter
+        font.family: "Helvetica"
+        font.pointSize: 10
+        color: "black"
+        text: leftButton.text
+      }
+    }
+    Button {
+      id: rightButton
+      text: "\u25B6"
+      checkable: true
+      Layout.row: 1
+      Layout.column: 2
+      onClicked: {
+        Teleop.angularDir = rightButton.checked ? -1 : 0
+        if(leftButton.checked)
+          leftButton.checked = false
+        Teleop.OnTeleopTwist()
+      }
+      Material.background: Material.primary
+      contentItem: Label {
+        renderType: Text.NativeRendering
+        verticalAlignment: Text.AlignVCenter
+        horizontalAlignment: Text.AlignHCenter
+        font.family: "Helvetica"
+        font.pointSize: 10
+        color: "black"
+        text: rightButton.text
+      }
+    }
+    Button {
+      id: backwardButton
+      text: "\u25BC"
+      checkable: true
+      Layout.row: 2
+      Layout.column: 1
+      onClicked: {
+        Teleop.linearDir = backwardButton.checked ? -1 : 0
+        if(forwardButton.checked)
+          forwardButton.checked = false
+        Teleop.OnTeleopTwist()
+      }
+      Material.background: Material.primary
+      contentItem: Label {
+        renderType: Text.NativeRendering
+        verticalAlignment: Text.AlignVCenter
+        horizontalAlignment: Text.AlignHCenter
+        font.family: "Helvetica"
+        font.pointSize: 10
+        color: "black"
+        text: backwardButton.text
+      }
+    }
+    Button {
+      id: stopButton
+      text: "Stop"
+      checkable: false
+      Layout.row: 1
+      Layout.column: 1
+      onClicked: {
+        Teleop.linearDir = 0
+        Teleop.angularDir = 0
+        forwardButton.checked = false
+        leftButton.checked = false
+        rightButton.checked = false
+        backwardButton.checked = false
+        linearVelSlider.value = 0
+        angularVelSlider.value = 0
+        slidersSwitch.checked = false
+        Teleop.OnTeleopTwist()
+      }
+      Material.background: Material.primary
+      contentItem: Label {
+        renderType: Text.NativeRendering
+        verticalAlignment: Text.AlignVCenter
+        horizontalAlignment: Text.AlignHCenter
+        font.family: "Helvetica"
+        font.pointSize: 10
+        color: "black"
+        text: stopButton.text
+      }
+    }
+  }
+
+  //Keyboard's switch
+  Switch {
+    id: keySwitch
+    anchors.top: buttonsGrid.bottom
+    anchors.topMargin: 10
+    anchors.left: parent.left
+    anchors.leftMargin: 5
+    onClicked: {
+      forwardButton.checked = false
+      leftButton.checked = false
+      rightButton.checked = false
+      backwardButton.checked = false
+      Teleop.OnKeySwitch(checked);
+    }
+    ToolTip.visible: hovered
+    ToolTip.text: checked ? qsTr("Disable keyboard") : qsTr("Enable keyboard")
+  }
+  Label {
+    id: keyboardSwitchLabel
+    text: "Input from keyboard"
```

Comment:
How about mentioning WASD so users know what keys to use? I tried the arrows at first.

```suggestion
    text: "Input from keyboard (WASD)"
```

Reply:
Addressed